android-interop-testing/app/src/main/proto/io/grpc/android/integrationtest/messages.proto

```diff
@@ -32,7 +32,7 @@
 
 syntax = "proto3";
 
-package grpc.testing;
+package grpc.testing.androidintegration;
```

Comment:
The package name was wrong to begin with, and breaks internal proto db.  This change makes that wrongness more apparent.  It will have to be fixed in other places, but only because we have a ton of copies rather than one true copy.